File: POO/TP2/Practica Python/EJ11/Empresa.py
from Persona import Persona
class Empresa:
    __nombre: str
    __direccion: str
    __empleados: list

    def __init__(self, nombre,direccion) :
        self.__nombre= nombre
        self.__direccion= direccion
        self.__empleados = []
        
    def get_nombre(self):
        return self.__nombre
    
    def set_nombre(self, nombre):
        self.__nombre = nombre
    
    def get_direccion(self):
        return self.__direccion
    def set_direccion(self, direccion):
        self.__direccion = direccion
    
    def get_empleados(self):
        return self.__empleados
    
    def add_Empleado(self, empleado):
        self.__empleados.append(empleado)
        
    def get_cantidad_empleados(self):
        return len(self.__empleados)

File: POO/TP2/Practica Java/EJ12/Empresa.java
package EJ12;

import java.util.ArrayList;

public class Empresa {
    private String nombre;
    private String direccion;
    private ArrayList<Persona> empleados;

    public Empresa(String nombre, String direccion) {
        this.nombre = nombre;
        this.direccion = direccion;
        this.empleados = new ArrayList<Persona>();
    }

    public void addEmpleado(Persona persona) {
        this.empleados.add(persona);
    }

    public Integer getCantidadEmpleados() {
        return this.empleados.size();
    }

    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDireccion() {
        return this.direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public ArrayList<Persona> getEmpleados() {
        return this.empleados;
    }

    public void setEmpleados(ArrayList<Persona> empleados) {
        this.empleados = empleados;
    }

}
